x-pack/plugins/threat_intelligence/public/modules/timeline/components/add_to_timeline/styles.ts

```diff
@@ -5,11 +5,14 @@
  * 2.0.
  */
 
-export function mockUiSetting(key: string): string | undefined {
```

Comment:
very weird diff, is it just `git` incorrectly interpreting it as a rename of the file?

Reply:
yeah for some reason git thinks I've renamed the file. Let me try to delete and recreate it real quick...

Reply:
didn't fix this as I would have had to amend the commit instead of adding a new one...